lib/cn/vblock_builder.c

```diff
@@ -102,6 +102,7 @@ struct vblock_builder {
     off_t                      wbuf_off;
     unsigned int               wbuf_len;
     uint64_t                   vgroup;
+    uint64_t                   tot_wlen;
```

Comment:
add `// excludes footer len`
and maybe change to `tot_vlen` since `wlen` is used everywhere else to mean written len.

Reply:
Can do.  Not clear why @nabeelmmd chose wlen, Nabeel?

Reply:
"wlen", "alen" and "clen" have specific meanings.  See `struct kvset_stats`.

Reply:
Agreed, wlen is a bad choice.. tot_vlen works for me!

Reply:
Fixed.